agent/tests/test_lib_aws_sdk_php.c

```diff
@@ -458,6 +482,103 @@ static void test_nr_lib_aws_sdk_php_handle_version(void) {
   tlib_php_request_end();
 }
 
+#if ZEND_MODULE_API_NO >= ZEND_8_1_X_API_NO
+static void test_nr_lib_aws_sdk_php_lambda_invoke() {
+  tlib_php_engine_create("newrelic.cloud.aws.account_id=\"test_account\"");
```

Comment:
This is one way to set an NRINI value.
You could also set it manually by just doing:
NRINI(aws_account_id) = NULL;
or NRINI(aws_account_id) ="123456789012";
so it's easier to do various test cases.

Can you please add a test case where `aws_account_id` is `NULL` and where it is `""`?

Reply:
aws_account_id ini should never be `NULL`, as it defauls to `""` and requires a valid 12-length string to set. Added test for "" in [5c45a0e](https://github.com/newrelic/newrelic-php-agent/pull/1023/commits/5c45a0ef346dbb2bf7e1b28a8ed0de4cbabcbb3f)

Reply:
Turns out it will be null, rather than the default, when the 12-length requirement is violated. Added tests here [25475ef](https://github.com/newrelic/newrelic-php-agent/pull/1023/commits/25475efa60752f455c8ca6bf90d5ef8e15ed1cc0)

Reply:
Why not just make it `""` if the length requirement is violated with an else statement [here](https://github.com/newrelic/newrelic-php-agent/pull/1023/commits/5c45a0ef346dbb2bf7e1b28a8ed0de4cbabcbb3f#diff-07c39860ed983532dfdc2a9d3093475beb7bd749067607c324a1e7718cffd92fR1237)?

Reply:
I followed the design of other functions that check for string length, like our [license key check](https://github.com/newrelic/newrelic-php-agent/pull/1023/commits/5c45a0ef346dbb2bf7e1b28a8ed0de4cbabcbb3f#diff-07c39860ed983532dfdc2a9d3093475beb7bd749067607c324a1e7718cffd92fL1163). If there is no reason to follow this design, then I can switch it.

Reply:
Hmm, I don't have a strong opinion one way or the other on this.
If it's consistent with other string behavior, I'm okay with it.